share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -336,7 +336,10 @@ python-pr-build:
     include:
       - artifact: jobs_scratch_dir/cloud-ci-pipeline.yml
         job: python-pr-generate
-    strategy: depend
+    strategy: depends
```

Comment:
The extra `s` that got accidentally added here is invalid gitlab yaml syntax.

Reply:
Yep, just noticed this at the same time you did. The validation feature of GitLab was pretty helpful for this.